Source/WebKit/Shared/Cocoa/WebCoreArgumentCodersCocoa.serialization.in

```diff
@@ -113,6 +113,18 @@ headers: <pal/cocoa/PassKitSoftLink.h>
 class WebCore::PaymentMethod {
     [Precondition='PAL::isPassKitCoreFrameworkAvailable()', SecureCodingAllowed=[PAL::getPKPaymentMethodClass()], Validator='m_pkPaymentMethod'] RetainPtr<PKPaymentMethod> m_pkPaymentMethod;
 }
+
+class WebCore::Payment {
+    [Precondition='PAL::isPassKitCoreFrameworkAvailable()', SecureCodingAllowed=[PAL::getPKPaymentClass()], Validator='pkPayment'] RetainPtr<PKPayment> pkPayment();
```

Comment:
The validators are not equivalent to what was there before.  decoder.decodeWithAllowedClasses<T> returns an optional<RetainPtr<T>>.  Same with WebCore::PaymentMethod whose validator should be removed.  Null is ok, decode failure is not.